xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/WebHome.xml

```diff
@@ -29,20 +29,42 @@
   <parent>Main.WebHome</parent>
   <creator>xwiki:XWiki.Admin</creator>
   <author>xwiki:XWiki.Admin</author>
-  <customClass/>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
   <creationDate>1372190329000</creationDate>
   <date>1372190329000</date>
   <contentUpdateDate>1372190329000</contentUpdateDate>
   <version>1.1</version>
   <title/>
-  <defaultTemplate/>
-  <validationScript/>
   <comment/>
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.0</syntaxId>
   <hidden>false</hidden>
+  <content>{{velocity}}
```

Comment:
This is bad. There was no reason to change this page to add new translations and you broke XWIKI-15387 :(

It's now been fixed by https://jira.xwiki.org/browse/XWIKI-15604 (again).